services/api-server/src/simcore_service_api_server/db/repositories/api_keys.py

```diff
@@ -34,7 +35,9 @@ async def get_user_id(self, api_key: str, api_secret: str) -> Optional[int]:
 
     async def any_user_with_id(self, user_id: int) -> bool:
         # FIXME: shall identify api_key or api_secret instead
-        stmt = sa.select([tbl.api_keys.c.user_id,]).where(
-            tbl.api_keys.c.user_id == user_id
-        )
+        stmt = sa.select(
```

Comment:
maybe just one thing here. when my PR [here](https://github.com/ITISFoundation/osparc-simcore/pull/2245) and hopefully before yours ;) these needs to be adapted to acquire the connection first.

Reply:
done